pydicom/tests/test_dataset.py

```diff
@@ -10,7 +10,7 @@
 from pydicom.data import get_testdata_files
 from pydicom.dataelem import DataElement, RawDataElement
 from pydicom.dataset import Dataset, FileDataset
-from pydicom.dicomio import read_file
+from pydicom.dicomio import dcmread
 from pydicom.filebase import DicomBytesIO
```

Comment:
Shouldn't that be `from pydicom.filereader import dcmread` ?

Reply:
I could do that change now ... but thought we needed some more discussion before removing dicomio.  I think I am ready to do so, but some time back (well before the move to pydicom organization), we were trying to consolidate some of the io that way.  But it never really completed; I'm not sure it is worth keeping.  Certainly we could change the internal references like this one, and leave fully removing dicomio to another PR.

Reply:
Yes, the reference does not really matter as long as `dicomio` is not removed. That can wait.